package.json

```diff
@@ -10,7 +10,11 @@
     "test:browser": "standard && npm run deps && NODE_ENV=test zuul tests/browser/*",
     "test:browser:local": "standard && npm run deps && NODE_ENV=test zuul --local 8080 -- tests/browser/*",
     "test:cov": "npm run test:server:cov && npm run test:browser",
-    "test": "npm run test:server && npm run test:browser"
+    "test": "npm run test:server && npm run test:browser",
+    "start": "http-server",
```

Comment:
A relative path can be passed as the first argument to `http-server` to determine the location served:
```sh
$ http-server ../../
```